rental.rb

```diff
@@ -1,9 +1,12 @@
 class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
   def initialize(date, book, person)
     @date = date
     @book = book
     @rentee = person
+    person.rentals << self unless person.rentals.include? self
+    book.rentals << self unless book.rentals.include? self
```

Comment:
Great job @ha-manel 

- [ ] Kindly fix the error thrown by your app when a user wants to create a new rental entry
![rental](https://user-images.githubusercontent.com/55596970/180433823-b21730b8-919f-4bf3-bb43-5266f92ab6c8.png)

Reply:
@CollinsTatang Hello, thank you so much for the review. I see that you put the person's number as "1" instead of "0". I put the indexes on each row (ex: [Teacher 0]) so if you kindly put 0 instead of 1, the rental will be created correctly.